gnomad/variant_qc/evaluation.py

```diff
@@ -288,54 +288,77 @@ def compute_grouped_binned_ht(
 
 
 def compute_binned_truth_sample_concordance(
-    ht: hl.Table, binned_score_ht: hl.Table, n_bins: int = 100
+    ht: hl.Table,
+    binned_score_ht: hl.Table,
+    n_bins: int = 100,
+    add_bins: Dict[str, hl.expr.BooleanExpression] = {},
 ) -> hl.Table:
     """
     Determines the concordance (TP, FP, FN) between a truth sample within the callset and the samples truth data
     grouped by bins computed using `compute_quantile_bin`.
 
     .. note::
-
         The input 'ht` should contain three row fields:
             - score: value to use for quantile binning
             - GT: a CallExpression containing the genotype of the evaluation data for the sample
             - truth_GT: a CallExpression containing the genotype of the truth sample
-
         The input `binned_score_ht` should contain:
              - score: value used to bin the full callset
              - bin: the full callset quantile bin
 
+    'add_bins` can be used to add additional global and truth sample binning to the final binned truth sample
+    concordance HT. The keys in `add_bins` must be present in `indexed_binned_score_ht` and the values in `add_bins`
```

Comment:
this might be a tiny bit clearer if you change indexed_binned_score_ht to binned_score_ht, since the indexed ht is created within the function and not passed in/referenced elsewhere in the docstring (but not a really important change)